### DISCUSSION TITLE
How to share Projects on Qfield

### DISCUSSION THREAD

**Discussion by nikki-12345** [opengisch/QField#3746] [Q&A]
Hello!

I have projects that are packaged into zip files, I would like to share them with the staff in the field, and I am trying to find if I can upload the project to QField Cloud for them to open 80 miles away, could someone please advise? 

Thank you

**Comment by duboisAM** [comment]
I believe QField Cloud only works to share QField project files and not other type of files (pdf, zip, etc.). 
Link to doc - upload to Qfield Cloud: [https://docs.qfield.org/get-started/tutorials/get-started-qfc/#upload-a-project](https://docs.qfield.org/get-started/tutorials/get-started-qfc/#upload-a-project)
Example of what an uploaded project looks like:
<img width="661" alt="image" src="https://user-images.githubusercontent.com/64656876/208481412-2ca81744-838d-4163-959d-5a853df51846.png">


If you are looking for a way to share a zipped file, I believe you should go for OneDrive, GoogleDrive, DropBox or another cloud storage solution.

**Comment by suricactus** [comment]
In QFieldCloud the user is uploading the whole QGIS project and it's files, all self contained in a single directory. When downloading on QField, the project will be packaged and downloaded on the phone (basically, running QFieldSync is on QFieldCloud).

You can have `.pdf`, `.mp3`, even `.exe` files in your QFieldCloud project directory and these files will be on the cloud too, but QField will ignore them. This is helpful if you project contains additional data and can be easily shared within the team.

QFieldCloud does not allow sharing individual files, it's either all the project files, or none.
